perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -17,6 +17,7 @@
 
 import javax.sql.DataSource;
 
+import cz.metacentrum.perun.core.api.exceptions.*;
```

Comment:
This is behavior of some IDEs. If there is no need to use .* in import, please just add specific class you need.
There it is probably "cz.metacentrum.perun.core.api.exceptions.IllegalArgumentException".